src/test/java/org/jenkinsci/plugins/GithubSecurityRealmTest.java

```diff
@@ -43,31 +43,35 @@ public void testEquals_true() {
+        GithubSecurityRealm d = new GithubSecurityRealm("http://jenkins.acme.com", "http://jenkins.acme.com/api/v3", "someid", "somesecret", "read:org", "example.com", false);
+        GithubSecurityRealm e = new GithubSecurityRealm("http://jenkins.acme.com", "http://jenkins.acme.com/api/v3", "someid", "somesecret", "read:org", "example.com", false);
         assertTrue(a.equals(b));
         assertTrue(b.equals(c));
         assertTrue(c.equals(b));
-        assertTrue(c.equals(d));
+        assertTrue(d.equals(e));
+        assertTrue(e.equals(d));
     }
 
     @Test
     public void testEquals_false() {
-        GithubSecurityRealm a = new GithubSecurityRealm("http://jenkins.acme.com", "http://jenkins.acme.com/api/v3", "someid", "somesecret", "read:org");
-        GithubSecurityRealm b = new GithubSecurityRealm("http://jenkins.acme.com", "http://jenkins.acme.com/api/v3", "someid", "somesecret", "read:org,repo");
-        GithubSecurityRealm c = new GithubSecurityRealm("http://jenkins.acme.com", "http://jenkins.acme.com/api/v3", "someid", "somesecret", "read:org", "", false);
-        GithubSecurityRealm d = new GithubSecurityRealm("http://jenkins.acme.com", "http://jenkins.acme.com/api/v3", "someid", "somesecret", "read:org,repo", "", false);
+        GithubSecurityRealm a = new GithubSecurityRealm("http:jenkins.acme.com", "http://jenkins.acme.com/api/v3", "someid", "somesecret", "read:org");
+        GithubSecurityRealm b = new GithubSecurityRealm("http:jenkins.acme.com", "http://jenkins.acme.com/api/v3", "someid", "somesecret", "read:org,repo");
+        GithubSecurityRealm c = new GithubSecurityRealm("http:jenkins.acme.com", "http://jenkins.acme.com/api/v3", "someid", "somesecret", "read:org", "", false);
+        GithubSecurityRealm d = new GithubSecurityRealm("http:jenkins.acme.com", "http://jenkins.acme.com/api/v3", "someid", "somesecret", "read:org,repo", "", false);
+        GithubSecurityRealm e = new GithubSecurityRealm("http:jenkins.acme.com", "http://jenkins.acme.com/api/v3", "someid", "somesecret", "read:org,repo", "example.com", false);
         assertFalse(a.equals(b));
         assertFalse(a.equals(d));
         assertFalse(a.equals(""));
         assertFalse(c.equals(b));
         assertFalse(c.equals(d));
         assertFalse(c.equals(""));
+        assertFalse(d.equals(e));
     }
 
     @Test
     public void testHasScope_true() {
-        GithubSecurityRealm a = new GithubSecurityRealm("http://jenkins.acme.com", "http://jenkins.acme.com/api/v3", "someid", "somesecret", "read:org,user,user:email", "", false);
-        GithubSecurityRealm b = new GithubSecurityRealm("http://jenkins.acme.com", "http://jenkins.acme.com/api/v3", "someid", "somesecret", "read:org,user,user:email");
+        GithubSecurityRealm a = new GithubSecurityRealm("http:jenkins.acme.com", "http://jenkins.acme.com/api/v3", "someid", "somesecret", "read:org,user,user:email");
```

Comment:
Did you mean to remove `//` from `http://`?  This comment applies to other parts as well.

Reply:
Thanks for catching that. It was an accident. I had commented out a bunch of lines while debugging and used a regex to remove the comments.